containers/python/pytorch_container_entry.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env sh
```

Comment:
Modify the permissions of this script such that it can be executed (like we did during the last meeting) and commit the permissions change.

Reply:
I‘m not sure what should I modify here,I just do the same thing as all other .sh file,do you mean I need change "#!/usr/bin/env sh" to " #!/bin/bash"?

Reply:
Ah, by permissions change I mean `chmod` the script so that the container can execute it. `777` should work just fine